src/ComparatorAndCreate.java

```diff
@@ -0,0 +1,34 @@
+class ComparatorAndCreate {
+    int countComparator(int[] array) {
```

Comment:
Liczenie porównań powinno być częścią algorytmu sortowania. Jeśli teraz zmienisz algorytm to liczba porównań nie ulegnie zmianie.